src/shrutiDJun20Assignments/ASCII.java

```diff
@@ -0,0 +1,22 @@
+package shrutiDJun20Assignments;
+
+/*
+ Input: J7u9y8h1h8j4b7j3j6
+ Ouput: Sum of all numbers in String is - 54.
+ */
+public class ASCII {
+
+	void findSum(String input) {
+		int sum = 0;
+		for (int index = 0; index < input.length(); index++) {
+			if (input.charAt(index) >= 48 && input.charAt(index) <= 57) {
+				sum += Integer.parseInt(String.valueOf(input.charAt(index)));
```

Comment:
sum += (str.charAt(index) - '0'); You can use this as well. 
Or  sum = sum + Character.getNumericValue(str.charAt(index);